Adds the REQUIRED flag to find_package for OpenMP so that
we can be sure in the conanfile that if with_openmp is set
that OpenMP is indeed used. This is important since we need to add
compiler flags in package_info() in this case.
--- CMakeLists.txt	
+++ CMakeLists.txt
@@ -105,7 +105,7 @@ if (${BUILD_EXAMPLES})
 endif ()
 
 if (WITH_OPENMP)
-  find_package (OpenMP)
+  find_package (OpenMP REQUIRED)
   if (OPENMP_FOUND)
     set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
     set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
